server/controllers/RequestController.js

```diff
@@ -0,0 +1,32 @@
+import { requests } from '../dummydatabase/dummydatabase';
+/**
+  * @class RequestController
+  * @description CRUD operations on Requests
+  */
+class RequestController {
+  /**
+   * @static
+    * @param {object} req - The request payload sent to the router
+    * @param {object} res - The response payload sent back from the controller
+    * @returns {object} - status Message and the all requests that belongs to a user
+    * @description This method gets a request by id in maintenance tracker
+    * @memberOf RequestController
+    */
+  static getAllRequests(req, res) {
+    if (requests.length === 0) {
+      return res.status(200).json({
+        status: 'fail',
+        data: {
+          message: 'Request not found!'
+        }
+      });
+    }
+    return res.status(200).json({
+      status: 'success',
+      data: requests[0]
```

Comment:
Since you are returning the whole requests why are you only returning the first member of the array?

Reply:
Request only has a single item in it